### ISSUE TITLE
GetTypeMissUseInspector says "resource (closed)" is not a valid return value of gettype

### ISSUE THREAD

**Issue by ntzm** [kalessil/phpinspectionsea#993]
| Subject    | Details                                                  |
| :--------- | :------------------------------------------------------- |
| Issue type | Bug     |
| IDE        | PhpStorm 2018.2.1 EAP Build #PS-182.3911.19, built on August 1, 2018                 |
| Plugin     | Php Inspections EA Extended, 3.0.5.1 |

Description or screenshot:
--

```php
$resource = fopen(__FILE__);
fclose($resource);

var_dump(gettype($resource) === 'resource (closed)');
```

https://3v4l.org/GvUnm

Environment details
---

I am running PHP 7.2

**Comment by funivan** [comment]
http://php.net/manual/en/function.gettype.php
```
Possible values for the returned string are:
"resource (closed)" as of PHP 7.2.0
```

**Comment by kalessil** [comment]
Fixed
